umbraco-heartcore/api-documentation/content-management/media/README.md

```diff
@@ -477,10 +477,14 @@ Get a list of children (media items) by their parent GUID ID.
 
 Create a new media item and optionally upload a file to the created item.
 
-Media would typically contain an upload field (the `Image` and `File` media types has this by default), which means it is possible to send a file along with the request to create a new media item. This is done by sending a multi-part request with the JSON body and the file. If the media item does not contain a file you can send a regular JSON request to create the media.
+Media typically contains an upload field (the `Image` and `File` media types has this by default). An upload field allows sending a file along with the request to create a new media item. This is done by sending a multi-part request with the JSON body and the file. The first MultipartBoundary contains the JSON body describing the content for the image. The second MultipartBoundary contains the file. If the media item does not contain a file you can send a regular JSON request to create the media.
```

Comment:
This part is a bit outdated now, as the default is an Image Cropper and not an Upload field. The File media type uses an Upload field as far as I remember, but Image will use Image Cropper by default. So I think we need to reword this a little bit.

Reply:
I removed some bits, and reworded others. Can you have a second look? :-)